var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -220,6 +219,14 @@ class Paraview(CMakePackage, CudaPackage):
     # https://gitlab.kitware.com/vtk/vtk/-/merge_requests/8653
     patch('vtk-adios2-module-no-kit.patch', when='@5.8:5.10')
 
+    @property
+    def generator(self):
+        # https://gitlab.kitware.com/paraview/paraview/-/issues/21223
+        if self.spec.satisfies('%xl') or self.spec.satisfies('%xl_r'):
```

Comment:
So is it okay for cce?

Reply:
Hi Chuck,

Sorry, I noted this in the last commit message but not in the ticket body. I can't reproduce my earlier failures with `cce`, so I've dropped it from the fix. If the problem resurfaces I'll document it in a separate issue, but for now I'd say we can limit this PR to just `xl` and `xl_r`.